### DISCUSSION TITLE
Kaggle 정복하기

### DISCUSSION THREAD

**Discussion by JeongJae-Private** [Pseudo-Lab/9th-builder#14] [plan]
## 프로젝트 소개

안녕하세요. 이번 스터디는 Data Scientist 로서 짧은 기간에 큰 성장을 이룰 수 있는 방법인 Kaggle에 대해 알아보고 실제로 프로젝트에 참여해 보고자 합니다.

### 프로젝트 목표
1. Kaggle에 대해 알아보기
2. 문제 해결 프로세스에 대해 알아보기
3. 머신러닝 부터 딥러닝 까지 각종 프로젝트를 진행해보기
4. 새로운 모델링 기법 트렌드를 배워보기

### 프로젝트 내용
<img width="200" alt="image" src="https://github.com/user-attachments/assets/13d60660-b4d5-4a6b-bd16-3d6417f74f2d">

- '머신러닝, 딥러닝 문제해결 전략' 책을 통해 Kaggle 에 입문해 보고자 합니다.

## 모집 대상 (⚠️중급 이상)  
Kaggle에 관심이 많으신 분
머신러닝 및 딥러닝에 대해 기본적인 지식이 있으신 분
PyTorch 를 사용하실 수 있는 분
적극적으로 스터디에 참여할 수 있으신 분
⚠️**수행해본 딥러닝 or 머신러닝 경험이 있다면 반드시 적어주시기를 부탁 드립니다!!**
⚠️**함께 오랫동안 학습하고, 인연을 만들어 가실분을 기다리고 있습니다!!**

**모집 인원**
- 2-5명 (기존 인원 제외)

### 첫 모임 날짜 & 시간 & 장소
- 날짜: 9월 11일(수) (일정은 변경 될 수 있으나, 매주 수요일 스터디 진행 예정입니다.) 
- 시간: 21시 30분
- 장소: Discord (자세한 사항은 추후 공지)

### 학습 자료
'머신러닝, 딥러닝 문제해결 전략' (신백균 지음) 책을 통해 기본 전략 및 지식을 습득 한 뒤, 실제 프로젝트에 참여할 예정입니다.
(프로젝트 참여 방법 및 팀 구성은 추후 공지 예정)

### 스터디 활동 방법
💡 **오픈 아카데미** 형태(=청강 가능)로 진행하며, 발표자료는 가짜연구소를 통해 **공개적으로 배포할** 수 있도록 할 예정입니다.
    
1. **진행 & 발표**
    - 출석체크를 위해 실명으로 참여 부탁드립니다.
    - 온라인에서 여러분을 만나는 환경이므로 웹캠을 꼭 준비해 주세요~
    - 발표자는 OT 날 선정 (참여 인원에 따라 1인 1-2회 정도 발표 예상)
    - 가능하다면 학습한 내용을 **Jupyter Book** 형태로 정리하고 싶습니다. 
    
2. **학습 활동 (필수)**
    - 주차별 학습내용 Jupyter Book 형태로 정리
    - **학습 내용에 대한 질문 및 공유**

## 주차별 목표와 학습 내용
💡 주차별 진행계획은 러너(스터디원)분들과 협의에 의해 변경될 수 있습니다.

| 날짜 | 내용 | 발표자 | 매체 | 
| ----- | ----- | -------- | ----- |
| 2024/09/11 | OT (스터디 소개, 방향 논의, Ice breaking)| [성정재]() | [링크]() | 
| 2024/09/25 | 추후 업데이트 예정 | [성정재]() | [링크]() |

**Comment by thrcle** [comment]
문제해결전략이라... 흥미로워보이는 제목이네요 화이팅입니다!

**Comment by WongiPark0628** [comment]
기대됩니다..!

**Reply by lynnminn** [reply]
강화학습 스터디 하실 때부터 응원했었는데, 요번엔 kaggle 이라니 엄청 재밌어보이네요 ㅎㅎ 읽어본적은 없지만.. 선정하신 책도 제목부터 굉장히 매력적이게 보입니다!

**Reply by JeongJae-Private** [reply]
앗 너무 감사드립니다! 처음 입문을 예린님 통해서 시작했는데 정말  많이 배우고 있습니다! 이번 강화학습 프로젝트도 너무 흥미로워 보이네요~!
